tests/UI/scripts/generate-jsdoc.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# To print only errors
+set +x
+
+DOC_DIRECTORY=".doc"
```

Comment:
You should use a `cd` command to this current directory otherwise this directory will be created depending on where you call it